pkg/apis/kops/componentconfig.go

```diff
@@ -116,6 +116,9 @@ type KubeletConfigSpec struct {
 	// image garbage collection is never run. Lowest disk usage to garbage
 	// collect to.
 	ImageGCLowThresholdPercent *int32 `json:"imageGCLowThresholdPercent,omitempty" flag:"image-gc-low-threshold"`
+	// imagePullProgressDeadline is the timeout for image pulls
```

Comment:
s/imagePullProgressDeadline/ImagePullProgressDeadline/
you could probably drop the newline as well

Reply:
Right.